ilastik/applets/tracking/structured/opStructuredTracking.py

```diff
@@ -264,6 +287,18 @@ def _runStructuredLearning(self,
                                             # print "[structuredTrackingGui] Increasing max nearest neighbors! LABELS/MERGERS t:{} id:{}".format(time-1, int(previous_label))
                                             break
 
+                                    if type[0] in ["FIRST", "SINGLETON(FIRST_LAST)"] and time in self.appearances.keys() and label in self.appearances[time].keys() and track in self.appearances[time][label].keys() and self.appearances[time][label][track]:
+                                        # print("---> appearance",time,label,track)
+                                        if (time, int(label)) in list(hypothesesGraph._graph.node.keys()):
+                                            hypothesesGraph._graph.node[(time, int(label))]['appearance'] = True
+                                            logger.info("[structuredTrackingGui] APPEARANCE: {} {}".format(time, int(label)))
```

Comment:
Please downgrade this to `logger.debug` as it is spamming the console and shouldn't be seen by users.